examples/jsm/nodes/transpiler/ShaderToyDecoder.js

```diff
@@ -0,0 +1,63 @@
+import { Return, VariableDeclaration, Accessor } from './AST.js';
+import GLSLDecoder from './GLSLDecoder.js';
+
+const keywords = [
+	{ name: 'iTime', polyfill: 'float iTime = timerLocal();' },
+	{ name: 'iResolution', polyfill: 'vec3 iResolution = viewportResolution;' },
+	{ name: 'fragCoord', polyfill: 'vec2 fragCoord = viewportCoordinate.setY( viewportCoordinate.y.oneMinus() ).toVar();' }
```

Comment:
We can also do `fragCoord = viewportCoordinate.toVar(); fragCoord.y.oneMinusAssign()`.